src/ucp/core/ucp_ep.c

```diff
@@ -1422,28 +1414,25 @@ ucs_status_t ucp_ep_config_init(ucp_worker_h worker, ucp_ep_config_t *config,
     config->tag.lane                    = UCP_NULL_LANE;
     config->tag.proto                   = &ucp_tag_eager_proto;
     config->tag.sync_proto              = &ucp_tag_eager_sync_proto;
-    config->tag.rndv.rma_thresh         = SIZE_MAX;
-    config->tag.rndv.min_get_zcopy      = 0;
-    config->tag.rndv.max_get_zcopy      = SIZE_MAX;
-    config->tag.rndv.min_put_zcopy      = 0;
-    config->tag.rndv.max_put_zcopy      = SIZE_MAX;
-    config->tag.rndv.am_thresh          = SIZE_MAX;
-    config->tag.rndv_send_nbr.am_thresh = SIZE_MAX;
-    config->tag.rndv_send_nbr.rma_thresh = SIZE_MAX;
-    config->tag.rndv.rkey_size          = ucp_rkey_packed_size(context,
+    config->tag.rndv.rma_thresh.remote  = SIZE_MAX;
+    config->tag.rndv.am_thresh.remote   = SIZE_MAX;
+    config->tag.rndv.rma_thresh.local   = SIZE_MAX;
+    config->tag.rndv.am_thresh.local    = SIZE_MAX;
+    config->rndv.min_get_zcopy          = 0;
+    config->rndv.max_get_zcopy          = SIZE_MAX;
+    config->rndv.min_put_zcopy          = 0;
+    config->rndv.max_put_zcopy          = SIZE_MAX;
+    config->rndv.rkey_size              = ucp_rkey_packed_size(context,
```

Comment:
should we init the following fields of EP config here:
```
config->rndv.am_thresh  = SIZE_MAX;
config->rndv.rma_thresh = SIZE_MAX;
```
?
or we really that they are initialized in 1613-1630 lines?

Reply:
yes